distribution/options.html

```diff
@@ -3,6 +3,13 @@
 <title>Refined GitHub options</title>
 <link rel="stylesheet" href="build/options.css">
 <form id="options-form" class="detail-view-container">
+	<p>
+		<strong>Hello!</strong><br>
```

Comment:
Not convinced about this, but also it felt weird as the only section without a title